lib/matplotlib/tests/test_figure.py

```diff
@@ -533,3 +534,17 @@ def test_removed_axis():
     fig, axs = plt.subplots(2, sharex=True)
     axs[0].remove()
     fig.canvas.draw()
+
+
+@pytest.mark.style('mpl20')
+def test_picking_staleness():
```

Comment:
Can you make this clearer? Either `test_picking_does_not_stale_figure()`, or add a docstring.